contrib/dotzlib/DotZLib/ChecksumImpl.cs

```diff
@@ -2,7 +2,7 @@
 // � Copyright Henrik Ravn 2004
 //
 // Use, modification and distribution are subject to the Boost Software License, Version 1.0.
-// (See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+// (See accompanying file LICENSE_1_0.txt or copy at https://www.boost.org/LICENSE_1_0.txt)

```

Comment:
Perhaps I shouldn't update the Boost standard licensing text, as it hasn't been updated in Boost sources.

Reply:
It's all fine as is, see question 1 in the FAQs of https://www.boost.org/users/license.html.